Source/utils/file_util.h

```diff
@@ -10,6 +10,7 @@
 namespace devilution {
 
 bool FileExists(const char *path);
+bool IsWriteable(const char *path);

```

Comment:
Perhaps we should rename this to `FileExistsAndIsWriteable` to better reflect what this checks for?